packages/block/src/types.ts

```diff
@@ -168,12 +177,26 @@ export type BlockBytes =
       WithdrawalsBytes,
       ExecutionWitnessBytes
     ]
+  | [BlockHeaderBytes, TransactionsBytes, UncleHeadersBytes, WithdrawalsBytes, DepositsBytes]
+  | [
+      BlockHeaderBytes,
+      TransactionsBytes,
+      UncleHeadersBytes,
+      WithdrawalsBytes,
+      DepositsBytes,
```

Comment:
Not sure if this is going to be before or after `ExecutionWitnessBytes` in Prague.

Reply:
ExecutionWitness is for Verkle which is not scheduled for Prague, so Deposits can always be placed before ExecutionWitness.